sample/Pizzeria.Store.Web/Layout/MainLayout.razor.cs

```diff
@@ -1,7 +1,9 @@
+using System.Diagnostics.CodeAnalysis;
 using Microsoft.AspNetCore.Components;
 
 namespace Pizzeria.Store.Web.Layout;
 
+[SuppressMessage("Performance", "CA1515:Consider making public types internal", Justification = "Blazor component code-behind must be public to match generated partial class")]
```

Comment:
This suppression shouldn't be needed as there's section in `.editorconfig` for `*.razor.cs` and this rule is turned off